apps/api/src/api.controller.ts

```diff
@@ -17,4 +19,76 @@ export class ApiController {
       status: HttpStatus.OK,
     };
   }
+
+  @Put('asset/:userDsnpId/upload')
```

Comment:
this is `PUT` since I'm imagining it to have idempotent behavior for the same content but there are arguments to be made for making it `POST` also.

Reply:
I've mostly given up in the PUT vs POST distinction. I think it is fine as a put.